{"name": "Verify login with invalid credentials", "status": "passed", "start": 1751975884689, "stop": 1751975886067, "uuid": "06f0f4e6-35ae-47f3-98d8-8a5ae5d43a80", "historyId": "0b4a530345edabbd36260c6e7a3a603d", "testCaseId": "0b4a530345edabbd36260c6e7a3a603d", "fullName": "tests.api.api_test#test_login_invalid_api", "labels": [{"name": "suite", "value": "API Tests"}, {"name": "parentSuite", "value": "tests.api"}, {"name": "host", "value": "WS01797"}, {"name": "thread", "value": "26996-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.api.api_test"}]}